.github/workflows/ci_workflows.yml

```diff
@@ -125,3 +125,22 @@ jobs:
       test_command: pytest -p no:warnings --astropy-header -m "not hypothesis" -k "not test_data_out_of_range and not test_set_locale and not TestQuantityTyping" --pyargs astropy
       targets: |
         - cp311-manylinux_x86_64
+
+  test_arm64:
```

Comment:
now that this runs in the main matrix, you may want to consider removing one of the other linux jobs?

Reply:
Any suggestion? It still needs to be a separate job due to different env var, I think.

All the existing Linux jobs seem to be specialized (all deps, cov, run twice, etc). And I am not sure which one is most suited to use this new runner.

Reply:
> It still needs to be a separate job due to different env var, I think.

It shouldn't you can set per-job env vars: https://github-actions-workflows.openastronomy.org/en/latest/tox.html#setenv

Reply:
Why not let the arm job do the run-twice or so? That would seem unlikely to be architecture dependent.

Reply:
I have to check the runner specs again. I think we have to reduce `-n=4` to `-n=2` if we switch. Is that a problem?

Reply:
Oh nvm... maybe we get 4 still. I must be confused with the paid one. https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories